utils/soca/gdas_soca_diagb.h

```diff
@@ -89,13 +89,11 @@ namespace gdasapp {
       oops::Variables vars(fullConfig, "variables.name");
       diagBConfig.socaVars = vars;
 
-      // Get the max std dev for ssh
-      diagBConfig.sshMax = 0.0;
-      fullConfig.get("max ssh", diagBConfig.sshMax);
+      // Get the max std dev for unbalanced ssh
+      diagBConfig.sshMax = fullConfig.getDouble("max ssh", 0.0);
 
-      // Get the minimum depth for which to partition the 3D field's std. dev.
-      diagBConfig.depthMin = 0.0;
-      fullConfig.get("min depth", diagBConfig.depthMin);
+      // Get the minimum depth in meters for which to partition the 3D field's std. dev.
+      diagBConfig.depthMin = fullConfig.getDouble("min depth", 500.0);
```

Comment:
Is there a good reason not to make the second parameter configurable somewhere, or at least a const? Same question for a number of lines in here.

Reply:
Not sure I understand your question or what "second parameter" is @AndrewEichmann-NOAA . In the lines you highlighted,  `diagBConfig.depthMin` comes from a configuration called `min depth`. It's initialized to `500.0m` if the config is not found.

Reply:
So it's a default? Ok